### PR TITLE
gnome-radio: update to 0.4.0

### PR DESCRIPTION
**Pull request by oleaamot**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.5.1 20G80 arm64
Xcode 12.5.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: gnome/gnome-radio/Portfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                gnome-radio
-version             0.2.0
+version             0.4.0
 revision            1
```
Comment (ryandesign) [inline gnome/gnome-radio/Portfile:7]: When the version is increased, the revision should be set to 0.

**Comment by reneeotten** [comment]
ping @oleaamot - can you please make the requested changes so that the PR can be merged? Thanks!

File: gnome/gnome-radio/Portfile (hunk 1 above)
Comment (oleaamot) [reply gnome/gnome-radio/Portfile]: OK, added new pull request with the revision set to 0.

**Comment by reneeotten** [comment]
@oleaamot for future reference: we typically would ask you to squash these changes into one commit. I'll do that now while merging the PR - thanks for the update!

**Pull request merged**